firestore.rules

```diff
@@ -48,13 +52,13 @@ service cloud.firestore {
       allow read: if request.auth.uid != null;
       allow write: if true;
     }
+    match /{path=**}/events/{event} {
+      allow read: if true;
+      allow write: if request.auth.uid in venueData().owners;
+    }
```

Comment:
NOTE: This needs to be explicitly set for groupQuery to work as per the docs
https://firebase.google.com/docs/firestore/security/rules-query#secure_and_query_documents_based_on_collection_groups